Update README.md (#36)
redhat-developer/rhdh-local: README.md

1. (Optional) The node `npx` tool (if you intend to use GitHub authentication in RHDH)
1. (Optional) A [Red Hat account](https://access.redhat.com/RegistryAuthentication#getting-a-red-hat-login-2) (if you want to use a PostgreSQL database)

> ### Note

Reviewer: I feel like this note would make more sense under the [Changing the Container Image](https://github.com/rrbanda/rhdh-local/blob/arm/README.md#changing-the-container-image) section. WDYT?

   For Mac M1 users, If you're using an Apple Silicon (M1/M2) Mac,

>    the default RHDH image (quay.io/rhdh/rhdh-hub-rhel9:1.4) is not compatible with the ARM64 architecture.To fix this, open your .env file and uncomment

Reviewer: ```suggestion
   the default RHDH image (quay.io/rhdh/rhdh-hub-rhel9:1.4) is not compatible with the ARM64 architecture. To fix this, open your `.env` file and uncomment
```

>    For Mac M1 users, If you're using an Apple Silicon (M1/M2) Mac,
>    the default RHDH image (quay.io/rhdh/rhdh-hub-rhel9:1.4) is not compatible with the ARM64 architecture. To fix this, open your `.env` file and uncomment
>    or replace the RHDH_IMAGE line with the following image that supports both amd64 and arm64: `RHDH_IMAGE=quay.io/rhdh-community/rhdh:next`

Reviewer: ```suggestion
   For Mac M1 users, if you're using an Apple Silicon (M1/M2) Mac, the default RHDH image (`quay.io/rhdh/rhdh-hub-rhel9:1.4`) is not compatible with the ARM64 architecture. To fix this, add this line to your `.env` file (create the file if it doesn't exist): `RHDH_IMAGE=quay.io/rhdh-community/rhdh:next`
   This image supports both `amd64` and `arm64`.
```

1. (Optional) The node `npx` tool (if you intend to use GitHub authentication in RHDH)
1. (Optional) A [Red Hat account](https://access.redhat.com/RegistryAuthentication#getting-a-red-hat-login-2) (if you want to use a PostgreSQL database)

### Note for Mac M1 users

If you're using an Apple Silicon (M1/M2) Mac, the default RHDH image (`quay.io/rhdh/rhdh-hub-rhel9:1.4`) is not compatible with the ARM64 architecture.
To fix this, you can add this line to your `.env` file (create the file if it doesn't exist):

> `RHDH_IMAGE=quay.io/rhdh-community/rhdh:next`

Reviewer: ```suggestion

`RHDH_IMAGE=quay.io/rhdh-community/rhdh:next`
```
